src/gpu/intel/ocl/reusable_softmax.cl

```diff
@@ -72,3 +74,140 @@ reusable_softmax_fwd_generic(__global SRC_DATA_T *src, __global DST_DATA_T *dst,
         dst[c - begin] = TO_DST(unscaled * scale);
     }
 }
+
+#endif
+
+#ifdef USE_VECTORIZED_KERNEL
+
+__attribute__((reqd_work_group_size(SUBGROUP_SIZE, 1, 1)))
+__attribute__((intel_reqd_sub_group_size(SUBGROUP_SIZE))) __kernel void
+reusable_softmax_fwd_generic(__global DATA_T *src, __global DST_DATA_T *dst,
+        __global float *src_scale, __global float *dst_scale,
+        dim_t SOFTMAX_AXIS_SIZE, dim_t softmax_axis_stride,
```

Comment:
nit: don't use capitalization for variable.

Reply:
This was supposed to be a draft PR. Sorry about that.